charts/fluent-bit/values.yaml

```diff
@@ -141,6 +141,39 @@ resources: {}
 #     cpu: 100m
 #     memory: 128Mi
 
+## only available if kind is Deployment
+autoscaling:
+  enabled: false
+#  minReplicas: 1
+#  maxReplicas: 100
+#  targetCPUUtilizationPercentage: 80
+#  targetMemoryUtilizationPercentage: 80
+   ## see https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics
+  customRules: []
+#     - type: Pods
+#       pods:
+#         metric:
+#           name: packets-per-second
+#         target:
+#           type: AverageValue
+#           averageValue: 1k
+    ## see https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-configurable-scaling-behavior
+#   behavior:
```

Comment:
```suggestion
  behavior: {}
```

This value is named in the template so should be uncommented and given an empty value.

Reply:
@stevehipwell check all changes now please